cmd/lakefs/cmd/run.go

```diff
@@ -61,6 +62,28 @@ type Shutter interface {
 	Shutdown(context.Context) error
 }
 
+func newExternalBasicAuthenticator(cfg *config.Config, service auth.Service, logger logging.Logger) (auth.Authenticator, error) {
+
+	if cfg.Auth.RemoteAuthenticator != nil && cfg.Auth.LDAP != nil {
+		return nil, fmt.Errorf("cannot configure both LDAP (Deprecated) and RemoteAuthenticator")
+	}
+
+	if cfg.Auth.RemoteAuthenticator != nil {
```

Comment:
I prefer to keep an _explicit_ `enabled` field.  For instance, it would allow me to disable the remote configuration by editing a single field, without having to delete it (which loses information) or comment it out (which is lots of work, and can be painful in YAML).

Reply:
Agree, done!